app/controllers/insured/family_members_controller.rb

```diff
@@ -48,6 +48,7 @@ def index
       broker_role_id = @consumer_role.person.broker_role.try(:id)
       @family.hire_broker_agency(broker_role_id)
     end
+    @family = Family.find(params[:family_id]) if params[:family_id]
```

Comment:
If we are not using any other details from the family object. can you only send the object id to the rendering files?

Reply:
Yeah

Reply:
# something like this
family = Family.where(id: params[:family_id])
@family_id = family.id if family.present?

Reply:
We ned it here cuz it will use @family later in the method